starter/clients/HttpClient.cs

```diff
@@ -0,0 +1,28 @@
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Spike.Inventory
```

Comment:
do we still want to name "spike" here? (I've mentioned this elsewhere)

Reply:
Ahhha... point of time... issue...

Reply:
Renamed Class, Function attribute and Namespace. Drops the word Spike from all places including project name.

New namespace - InventoryManagement